src/main/java/com/softlayer/api/http/HttpBasicAuthCredentials.java

```diff
@@ -10,4 +13,22 @@ public HttpBasicAuthCredentials(String username, String apiKey) {
         this.username = username;
         this.apiKey = apiKey;
     }
+
+    /**
+     * Gets the encoded representation of the basic authentication credentials
+     * for use in an HTTP Authorization header.
+     *
+     * @return String
+     * @throws UnsupportedEncodingException If encoding with UTF-8 fails.
+     */
+    public String getHeader() throws UnsupportedEncodingException
+    {
+        String authPair = username + ':' + apiKey;
+        String result = "Basic ";
+        result += new String(
```

Comment:
Consider `Base64.getEncoder().encodeToString` (none of the chars are unicode, doesn't matter what the charset is on the way back)

Reply:
Oh nice, much better

Reply:
Changed to use encodeToString and getBytes